### ISSUE TITLE
n("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.") two cam opened but one has the avatar and the other one could not find my face

### ISSUE THREAD

**Issue by inflamablez** [alievk/avatarify#288]
**Describe the bug**

**To Reproduce**

**Info (please complete the following information):**
 - OS (e.g., Linux): 
 - GPU model: 

 - Any other relevant information: 

**Screenshots**

**Logs**

**Comment by alievk** [comment]
Please re-open an issue properly filling the subject and the description.
